src/supernode/pom.xml

```diff
@@ -40,6 +40,12 @@
 
 
     <dependencies>
+        <dependency>
+           <groupId>org.mockito</groupId>
+           <artifactId>mockito-all</artifactId>
+           <version>1.10.19</version>
```

Comment:
How did you choose the version? Is it to match the one used in deeptest-utils? https://github.com/diffblue/deeptest-utils/blob/25a50b1c1c028b66ac03e98eb74376a4ad2f65f7/pom.xml#L68

Reply:
I used the one from platform docs, it's the same version, so we should be fine ;)